src/test/run-pass/align-struct.rs

```diff
@@ -176,6 +182,18 @@ pub fn main() {
     }
     assert!(is_aligned_to(&e, 16));
 
+    // check union alignment
+    assert_eq!(mem::align_of::<UnionContainsAlign>(), 16);
+    assert_eq!(mem::size_of::<UnionContainsAlign>(), 16);
```

Comment:
I translated this to the equivalent in VC++ and it reports the size and alignment being 16.